freqtrade/fiat_convert.py

```diff
@@ -0,0 +1,161 @@
+import logging
+import time
+from pymarketcap import Pymarketcap
+
+logger = logging.getLogger(__name__)
+
+
+class PairConvertion():
+    # Constants
+    CACHE_DURATION = 6 * 60 * 60  # 6 hours
+
+    # Private attributes
+    crypto_symbol = None
```

Comment:
Instances variables should only be created within `__init__()` otherwise it could lead to confusion and it consumes unneeded memory.
Please also use an [underscore prefix](https://docs.python.org/3.6/tutorial/classes.html#private-variables) to highlight private attributes.

Reply:
Updated